### PR TITLE
Add blank participant_id check

### PR DESCRIPTION
**Pull request by acovrig**
## Summary
- Check veteran for `participant_id`
- Return a 422 with a helpful message

## Related issue(s)
- [API-22981](https://vajira.max.gov/browse/API-22981)

## Testing done
- rspec test written

## What areas of the site does it impact?
* *Note*: This may have a merge conflict with [another PR](https://github.com/department-of-veterans-affairs/vets-api/pull/11635) (specifically modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb)
* vets-api v2

## Acceptance criteria
- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by mchristiansonVA — APPROVED** [review]
LGTM

File: modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb [hunk 1]
```diff
@@ -9,6 +9,27 @@
...
+            ssn: '777000000',
+            icn: '1013062086V794840',
+            edipi: '1013590059',
+            participant_id: nil,
+            birls_id: '796104437'
+          }
+        )
+      }
+    )
+  end
```
Comment (FonzMP) [inline modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb:32]: can we not use the [mvi profile](https://github.com/department-of-veterans-affairs/vets-api/blob/e70358727877c5061b7f50328d9320c8c9797b0a/spec/factories/mvi_profiles.rb#L38) factory? Not sure if it has everything but could be as simple as `build(:mpi_profile)` and then set the `profile[:status] = nil` in the test.

**Review by mchristiansonVA — APPROVED** [review]
Looks good.

**Review by FonzMP — APPROVED** [review]
:shipit:

**Pull request merged**